x-pack/heartbeat/monitors/browser/synthexec/execmultiplexer.go

```diff
@@ -25,18 +24,11 @@ func (e ExecMultiplexer) writeSynthEvent(se *SynthEvent) {
 	}
 
 	if se.Type == "journey/start" {
-		e.currentJourney.Store(true)
 		e.eventCounter.Store(-1)
 	}
-	hasCurrentJourney := e.currentJourney.Load()
-	if se.Type == "journey/end" || se.Type == "cmd/status" {
-		e.currentJourney.Store(false)
-	}
-
 	se.index = e.eventCounter.Inc()
-	if hasCurrentJourney {
-		e.synthEvents <- se
-	}
+
+	e.synthEvents <- se
```

Comment:
There's a weird thing to consider, which is errors between journeys.

We could still need to check for `journey/end` to account for errors that might happen between journeys. I think we can just set `e.currentJourney.store(false)` again, no? Then it just gets reported as an error on the suite. 

Our long term goal is for heartbeat to really only execute one journey per monitor (with discovery mode) so I'm OK with not handling this for the moment and just tacking those errors on the previous journey. Also, I don't know what an error between journeys would even be.

Reply:
I am not sure if there would be error between journeys, But with this change we would still get `stderr` messages on the suite level, wouldnt that be enough to figure out any errors?